osu.Game/Screens/Play/MasterGameplayClockContainer.cs

```diff
@@ -115,14 +115,14 @@ protected override void StartGameplayClock()
         /// <summary>
         /// Skip forward to the next valid skip point.
         /// </summary>
-        public void Skip()
+        public void Skip(bool fullLength = false)
```

Comment:
Might at least need to update xmldoc to say that the skip is to "next valid point" only if `fullLength == false`, and otherwise it's to the latest possible skip point.

Not a huge fan of the naming either but I can't come up with anything much better so probably fine as long as xmldoc elaborates.